docs/flow/reference/components/prometheus.scrape.md

```diff
@@ -140,15 +148,74 @@ components.
+endpoints using HTTP, with a scrape interval of 1 minute and scrape timeout of
+10 seconds. The metrics path, protocol scheme, scrape interval and timeout,
+query parameters, as well as any other settings can be configured using the
+component's arguments.
+
+The scrape job expects the metrics exposed by the endpoint to follow the
+[OpenMetrics](https://openmetrics.io/) format. All metrics are then propagated
+to each receiver listed in the component's `forward_to` argument.
+
+Labels coming from targets, that start with a double underscore `__` are
+treated as _internal_, and are removed prior to scraping.
+
+The `prometheus.scrape` component regards a scrape as successful if it
+responded with a 200 status code and returned a body of valid metrics.
+
+If the scrape request fails, the component's debug endpoint and debug UI
+section contains more detailed information about the failure, the last
+successful scrape, as well as the labels last used for scraping.
+
+The following labels are automatically injected to the scraped time series and
+can help pin down a scrape target.
+
+Label                 | Description
+--------------------- | ---------- 
+job                   | The configured job name that the target belongs to. Defaults to the fully formed component name.
+instance              | The `__address__` or `<host>:<port>` of the scrape target's URL.
+
+
+Similarly, these metrics that record the behavior of the scrape targets are
+also automatically available.
+Metric Name                | Description
+-------------------------- | -----------
+`up`                       | 1 if the instance is healthy and reachable, or 0 if the scrape failed.
+`scrape_duration_seconds`  | Duration of the scrape in seconds.
+`scrape_samples_scraped`   | The number of samples the target exposed.
+`scrape_samples_post_metric_relabeling` | The number of samples remaining after metric relabeling was applied.
+`scrape_series_added`      | The approximate number of new series in this scrape.
+`scrape_timeout_seconds`   | The configured scrape timeout for a target. Useful for measuring how close a target was to timing out using `scrape_duration_seconds / scrape_timeout_seconds`
+`scrape_sample_limit`      | The configured sample limit for a target. Useful for measuring how close a target was to reaching the sample limit using `scrape_samples_post_metric_relabeling / (scrape_sample_limit > 0)`
+`scrape_body_size_bytes`   | The uncompressed size of the most recent scrape response, if successful. Scrapes failing because the `body_size_limit` is exceeded report -1, other scrape failures report 0.
+
+
+The `up` metric is particularly useful for monitoring and alerting on the
+health of a scrape job. It is set to zero in case anything goes wrong with the
+scrape target, either because it is not reachable, because the connection
+times out while scraping, or because the samples from the target could not be
+processed.
```

Comment:
```suggestion
The `up` metric is particularly useful for monitoring and alerting on the
health of a scrape job. It is set to `0` in case anything goes wrong with the
scrape target, either because it is not reachable, because the connection
times out while scraping, or because the samples from the target could not be
processed. When the target is behaving normally, the `up` metric is set to
`1`.
```